### PR TITLE
PMM-8308 track process exec path

### PR DESCRIPTION
**Pull request by qwest812**
https://jira.percona.com/browse/PMM-8308
fb https://github.com/Percona-Lab/pmm-submodules/pull/2307
pmm https://github.com/percona/pmm/pull/821
pmm-admin https://github.com/percona/pmm-admin/pull/176
pmm-agent  https://github.com/percona/pmm-agent/pull/314

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ services:
       - 15432:5432
     volumes:
       - .:/root/go/src/github.com/percona/pmm-managed
+      - ../pmm:/root/go/src/github.com/percona/pmm
```
Comment (BupycHuk) [inline docker-compose.yml:37]: let's revert it, it might bring new problems.

File: models/database.go [hunk 2]
```diff
@@ -683,6 +683,10 @@ var databaseSchema = [][]string{
 		`ALTER TABLE ia_templates
 			DROP COLUMN tiers`,
 	},
+	57: {
+		`ALTER TABLE agents
+		ADD COLUMN process_exec_path VARCHAR`,
```
Comment (BupycHuk) [inline models/database.go:688]: please check that we won't have issues with long paths

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Please write a few tests. 
If we included showing exec path in front-end into the tasks then we should expose it in API.

File: docker-compose.yml (hunk 1 above)
Comment (qwest812) [reply docker-compose.yml]: I reverted it

File: services/converters.go [hunk 3]
```diff
@@ -216,13 +216,18 @@ func ToAPIAgent(q *reform.Querier, agent *models.Agent) (inventorypb.Agent, erro
 		}
 		serviceID = service.ServiceID
 	}
+	var agentProcess string
+	if agent.ProcessExecPath != nil {
+		agentProcess = *agent.ProcessExecPath
+	}
```
Comment (BupycHuk) [inline services/converters.go:222]: this block can be replaced by `pointer.GetString`

File: services/converters.go [hunk 4]
```diff
@@ -220,9 +220,10 @@ func ToAPIAgent(q *reform.Querier, agent *models.Agent) (inventorypb.Agent, erro
 	switch agent.AgentType {
 	case models.PMMAgentType:
 		return &inventorypb.PMMAgent{
-			AgentId:      agent.AgentID,
-			RunsOnNodeId: pointer.GetString(agent.RunsOnNodeID),
-			CustomLabels: labels,
+			AgentId:         agent.AgentID,
+			RunsOnNodeId:    pointer.GetString(agent.RunsOnNodeID),
+			CustomLabels:    labels,
+			ProcessExecPath: *agent.ProcessExecPath,
```
Comment (BupycHuk) [inline services/converters.go:226]: please use pointer lib to get value instead of dereference in all places

Comment (qwest812) [reply services/converters.go]: fixed

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**